src/test/scala/dx/compiler/CompilerTest.scala

```diff
@@ -957,6 +957,27 @@ class CompilerTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll {
     trainsOutputVector.outputVars.size shouldBe 1
   }
 
+  it should "compile workflow with task imported by multiple paths" taggedAs NativeTest in {
+    val path = pathFromBasename("compiler", basename = "multi_import.wdl")
+    val args = path.toString :: cFlags
+    val retval = Main.compile(args.toVector)
+    retval shouldBe a[Success]
+  }
+
+  it should "compile workflow with call to task having optional output" taggedAs NativeTest in {
+    val path = pathFromBasename("compiler", basename = "optional_call_output.wdl")
+    val args = path.toString :: cFlags
+    val retval = Main.compile(args.toVector)
+    retval shouldBe a[Success]
+  }
+
+  it should "compile myriad workflow" taggedAs NativeTest in {
```

Comment:
It looks like this was left here by mistake.

Reply:
oops - removed it